Shared/Samples/Show line of sight between geoelements/ShowLineOfSightBetweenGeoelementsView.swift

```diff
@@ -0,0 +1,335 @@
+            renderer.sceneProperties.headingExpression = "[HEADING]"
+            overlay.renderer = renderer
+            return overlay
+        }()
+        
+        /// Overlay used to display the line of sight analysis visualization.
+        let analysisOverlay = AnalysisOverlay()
+        
+        /// A line of sight analysis between the observer and the taxi graphic.
+        private let lineOfSight: GeoElementLineOfSight
+        
+        /// A graphic representing the taxi model that will be animated.
+        private let taxiGraphic: Graphic = {
+            let sceneSymbol = ModelSceneSymbol(url: .taxi)
+            sceneSymbol.anchorPosition = .bottom
+            let graphic = Graphic(
+                geometry: .taxiPoint,
+                symbol: sceneSymbol
+            )
+            return graphic
+        }()
+        
+        /// A graphic representing the observer's location in the scene.
+        private let observerGraphic = Graphic(
+            geometry: .observerPoint,
+            symbol: SimpleMarkerSceneSymbol(
+                style: .sphere,
+                color: .red,
+                height: 5,
+                width: 5,
+                depth: 5,
+                anchorPosition: .bottom
+            )
+        )
+        
+        private var displayLink: CADisplayLink!
+        
+        var visibilityStatus = ""
+        
+        init() {
+            graphicsOverlay.addGraphics([observerGraphic, taxiGraphic])
+            lineOfSight = GeoElementLineOfSight(observer: observerGraphic, target: taxiGraphic)
+            lineOfSight.targetOffsetZ = 2
+            analysisOverlay.addAnalysis(lineOfSight)
+        }
+        
+        func setupAnimation() {
+            displayLink = makeDisplayLink()
+            displayLink.isPaused = false
+        }
+        
+        /// Creates a display link timer for the image overlay animation.
+        /// - Returns: A new `CADisplayLink` object.
+        private func makeDisplayLink() -> CADisplayLink {
+            // Create new display link.
+            let newDisplayLink = CADisplayLink(
+                target: self,
+                selector: #selector(animateTaxi)
+            )
+            // Set the default frame rate to 60 fps.
+            newDisplayLink.preferredFramesPerSecond = 60
+            newDisplayLink.isPaused = true
+            // Add to main thread common mode run loop, so it is not effected by UI events.
+            newDisplayLink.add(to: .main, forMode: .common)
+            return newDisplayLink
+        }
+        
+        /// Used for deallocating `displayLink` on view disappear.
+        func stopAnimating() {
+            displayLink.invalidate()
+            displayLink = nil
+        }
+        
+        /// Animates the target graphic between a set of points in a loop,
+        /// updating the heading and visibility analysis on each frame.
+        @objc
+        private func animateTaxi() {
```

Comment:
```suggestion
        private func animateTaxiGraphic() {
```
This name seems more clear.